### ISSUE TITLE
[BUG] Sidebar hiding options not working as expected if it's collapsed

### ISSUE THREAD

**Issue by tautautautautau** [SevenTV/Extension#556]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest nightly version

- [X] I am using the latest nightly version

### What browsers are you seeing the problem on?

Chrome

### Current Behavior

When sidebar is open the settings "Hide Recommended Channels" and "Hide Viewers Also Watch" function correctly.

When sidebar is collapsed the "Hide Recommended Channels" hides "viewers also watch" and "Hide Viewers Also Watch" does nothing.

### Expected Behavior

When sidebar is collapsed the settings "Hide Recommended Channels" and "Hide Viewers Also Watch" function correctly.

### Steps To Reproduce

Change the mentioned settings when sidebar is collapsed.

**Comment by AnatoleAM** [comment]
related: #481, @ZemyCode

**Comment by Nevicar** [comment]
Is there perhaps an ETA on when this might make it into a nightly/main version? :o Been using the nightly version since the Kick release and haven't seen it pop up yet. Just swapped to the main version and still nothin.

**Comment by pimothyxd** [comment]
Should be fixed in #767

**Comment by Nevicar** [comment]
Yea I'm not sure, just reporting what I see. Here's an example. I am not running any other extensions beside 7tv. When everything is expanded, there's no issue.

![image](https://github.com/SevenTV/Extension/assets/5368799/a844fbc4-235a-4fab-be23-cfbf8cf3045a)

But when I collapse, it (the Recommended Channels section) comes back.

![image](https://github.com/SevenTV/Extension/assets/5368799/38f56005-7d60-47fd-9e87-f5fd940a0e94)

Recommended Channels never goes away when collapsed. "Hide Viewers Also Watch" in 7tv does nothing for me when I toggle it when collapsed - but when I toggle "Recommended Channels" in 7tv it gets rid of the Viewers Also Watch section when collapsed. Not sure if it's just me or what, but I'm pretty sure I've seen this behavior between both Chrome and Firefox.

**Comment by pimothyxd** [comment]
Yeah, the pull request hasn't actually been merged yet. So you'd have to wait for that first.
